automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -65,7 +63,7 @@ class PublishSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpe
                 // Micro-sleep to keep the test from failing (let Elasticsearch catch up?)
                 //            Thread sleep 500
 
-                retry[Boolean](100.milliseconds, 1.minute)({
+                Retry.retry[Boolean](100.milliseconds, 1.minute)({
```

Comment:
Intellij compile complained `retry` not found. I fixed compile error by adding `Retry`